pkg/router/template/router.go

```diff
@@ -1489,3 +1492,27 @@ func privateKeysFromPEM(pemCerts []byte) ([]byte, error) {
 	}
 	return buf.Bytes(), nil
 }
+
+// watchCABundleCert watches the directory containing the CA bundle certificate
+// and reloads the router if the directory contents change.
+func (r *templateRouter) watchCABundleCert() error {
+	caBundleDir := filepath.Dir(r.defaultDestinationCAPath)
+
+	reloadFn := func() {
+		if err := r.reloadRouter(false); err != nil {
```

Comment:
Is there a reason this doesn't use the `rateLimitedCommitFunction`, like other functions such as `watchMutualTLSCert` use?: https://github.com/openshift/router/blob/19d1b7ce0c9e52182542ff9e447a59846c6948af/pkg/router/template/router.go#L471

This ensures the router doesn't reload more than the `spec.tuningOptions.reloadInterval` in the ingress controller. Reloads can be quite disruptive. Do we have a good reason to not use this logic?

Reply:
suggestion incorporated.